src/main/java/com/developerphil/adbidea/ui/MyDeviceChooser.java

```diff
@@ -112,7 +112,7 @@ public MyDeviceChooser(boolean multipleSelection,
 
     myFacet = facet;
     myFilter = filter;
-    myMinSdkVersion = AndroidModuleInfo.getInstance(facet).getRuntimeMinSdkVersion();
+    myMinSdkVersion = AndroidModuleInfo.get(facet).getRuntimeMinSdkVersion();
```

Comment:
This should be reverted, otherwise it won't run on Android Studio 3.0.
The release is already bound to 3.0 and up, it won't affect anyone on 2.3 (they won't get the updated version)